backend/typescript/rest/entityRoutes.ts

```diff
@@ -0,0 +1,91 @@
+import express, { Router } from 'express';
+import { Entity } from "../models/entity.mgmodel"
+import EntityService from "../services/implementations/EntityService";
+import { EntityRequestDTO } from "../services/interfaces/IEntityService";
+
+export const entityRouter: Router = Router();
+
+/* Create entity Object */
+entityRouter.post("/", async (_req, res) => {
+    const entityObj: EntityRequestDTO = _req.body.entity;
+    const IentityObj = {} as Entity;
+    const entService = new EntityService();
```

Comment:
1 instance of `EntityService` can be shared by all the endpoints.

Also just curious, would this approach have worked?
https://www.toptal.com/express-js/nodejs-typescript-rest-api-pt-1